modules/customizing-cli-downloads.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * web-console/customizing-the-web-console.adoc
+
+[id="creating-custom-CLI-downloads_{context}"]
+= Customizing CLI downloads
+
+You can configure links for downloading the CLI with custom link text and URLs,
+which can point directly to file packages or to an external page that provides
+the packages.
+
+.Procedure
+
+. Navigate to *Administration* -> *Custom Resource Definitions*.
+
+. Select *ConsoleCLIDownload* from the list of Custom Resource Definitions (CRDs).
+
+. Click the *YAML* tab, and then make your edits.
```

Comment:
Isn't it that you have to create an *instance* of a `ConsoleCLIDownload` CRD and not *edit* the CRD itself ? This does not make sense to me ...